ui/src/shared/components/Notification.js

```diff
@@ -0,0 +1,93 @@
+import React, {Component, PropTypes} from 'react'
+import {connect} from 'react-redux'
+import {bindActionCreators} from 'redux'
+
+import classnames from 'classnames'
+
+import {dismissNotification as dismissNotificationAction} from 'shared/actions/notifications'
+
+class Notification extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      opacity: 1,
+      height: 0,
+      dismiss: false,
```

Comment:
i would expect the idiomacy of this state `dismiss` to be more declarative, rather than imperative